module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -509,6 +520,27 @@ public function init()
         $this->sessionCache = $factory($namespace);
     }
 
+    /**
+     * Establish INN-Reach database connection
+     *
+     * @return resource|void
```

Comment:
I think it would be best to use `?resource` here instead of `resource|void` for consistency with the property definition above (and brevity).